.secretlintrc.json

```diff
@@ -0,0 +1,23 @@
+{
+  "rules": [
+    { "id": "@secretlint/secretlint-rule-no-k8s-kind-secret" },
```

Comment:
For this repo, this rule is not needed.

Reply:
yeah good catch, was trying out the different plugins and forgot to remove this again, gone now